zerver/models.py

```diff
@@ -412,9 +412,16 @@ class Realm(models.Model):
     # Messages older than this message ID in the organization are inaccessible.
     first_visible_message_id: int = models.IntegerField(default=0)
 
-    # Valid org_types are {CORPORATE, COMMUNITY}
+    # Valid org_types
     CORPORATE = 1
     COMMUNITY = 2
+    OPEN_SOURCE = 3
+    ACADEMIC = 4
+    RESEARCH = 5
+    EVENT = 6
+    NON_PROFIT = 7
+    PERSONAL = 8
+    OTHER = 9
     org_type: int = models.PositiveSmallIntegerField(default=CORPORATE)
```

Comment:
We should avoid having duplicates of this list.  I think `GIPHY_RATING_OPTIONS`  are `BOT_CREATION_POLICY_TYPES` are good examples of how one can set this up; basically we define in `models.py` the set of options and their labels, and then other files can just import this (e.g. we can pass a dictionary as context to the HTML template).

@sahil839 might have more advice.

Reply:
I think an object/dict like `GIPHY_RATING_OPTIONS` would be better.